docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

```diff
@@ -24,4 +24,4 @@ If you want to enable device config backup on existing nodes in a requisition, s
 . Click *Return* and click *Synchronize*.
 . Choose a scan option and click *Synchronize*.
 
-NOTE: By specifying the value `never` for the key `schedule` device configuration backups will not by triggered automatically. Of course it is still possible to retrieve backups on demand by using the Web-UI or the ReST services.
+NOTE: By specifying the value `never` for the key `schedule` device configuration backups will not be triggered automatically. It is still possible to retrieve backups on demand by using the Web-UI or the REST services.
```

Comment:
Sorry, I don't know why it's not letting me quote the line directly. Please add comma after "schedule" and remove the hyphen in "Web-UI". 

NOTE: By specifying the value `never` for the key `schedule`, device configuration backups will not be triggered automatically. It is still possible to retrieve backups on demand by using the Web UI or the REST services.

Reply:
Sorry, I missed that. Should be fine now.